swift-signal-handle.rst

```diff
@@ -69,11 +70,15 @@ temporary URL created by the above SwiftSignalHandle resource.
                 # assume you are doing a long running operation here
                 sleep 300
 
-                # Assuming long running operation completed successfully, notify success signal
+                # Assuming long running operation completed successfully,
+                # notify success signal
                 wc_notify --data-binary '{"status": "SUCCESS"}'
 
               params:
-                wc_notify: { get_attr: ['signal_handle', 'curl_cli'] 
+                # 'curl-cli' option will replace 'wc_notify' in the bash script
```

Comment:
Could be clearer. Maybe something like:

  # Replace all occurances of "wc_notify" in the script with an appropriate curl 
  # PUT request using the "curl_cli" attribute of the SwiftSignalHandle